### ISSUE TITLE
Tests are failing on main `solc` integrity not matching

### ISSUE THREAD

**Issue by ryardley** [gnosisguild/enclave#49]
TLDR:
1. Merge conflicts between main and ciphernode_poc causes tests not to run on the draft PR
2. Tests are not running correctly on main so main cannot be merged to ciphernode_poc as we require tests to function

~Not super urgent but would be good to fix soon~ This is probably now urgent.

---

Attempting to merge main into `ciphernode_poc` it became apparent that tests were failing on `main` which means we cannot practice continuous integration correctly 

![image](https://github.com/user-attachments/assets/af6a94a7-1eff-4b43-a18b-4b22448c0561)

Some linting fixes were made to ciphernode_poc so that evm tests would pass as continuous integration requires all tests should be runnable at all times

Possibly a github bug (or feature) but the merge conflict appears to prevent the test suite running on merge to ciphernode_poc. (Green ticks are missing)

![image](https://github.com/user-attachments/assets/add0e903-fcb7-4551-a682-d6bd8b8c5ff7)

~Having said all this tests are still running on PRs to ciphernode_poc itself so this is NOT urgent~ Now as we get close to integration this is urgent as we need to merge main back to ciphernode poc.
